tests/ocsp-failure.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Connection with OCSP checks fails
+--SKIPIF--
+<?php
+if ($_ENV['TESTS'] !== 'tests/ocsp-failure.phpt') { echo "skip OCSP tests not wanted\n"; }
+?>
+--FILE--
+<?php
+require_once __DIR__ . "/utils/basic.inc";
+
+$ping = new \MongoDB\Driver\Command(['ping' => 1]);
+
+// Expect command to fail with the provided options
+$m = new \MongoDB\Driver\Manager(URI);
+echo throws (function () use ($m, $ping) {
+    $m->executeCommand('admin', $ping);
+}, "MongoDB\Driver\Exception\ConnectionTimeoutException"), "\n";
+
+// Always expect command to pass when using insecure option
+$m = new \MongoDB\Driver\Manager(URI, ['tlsInsecure' => true]);
+$m->executeCommand('admin', $ping);
+
+// Always expect command to pass when allowing invalid certificates
+$m = new \MongoDB\Driver\Manager(URI, ['tlsAllowInvalidCertificates' => true]);
+$m->executeCommand('admin', $ping);
```

Comment:
The absence of an exception is sufficient here, correct? Just curious if we should be inspecting the result document's `ok` field.

Reply:
Correct. If the connection is not successful, we'll get an exception. The ping command failing would point to a different problem unrelated to OCSP and would be caught by other tests as well.